module_tunnels/koupleless_mqtt_tunnel/mqtt_tunnel.go

```
@@ -56,7 +57,7 @@
 	return utils.GetBizIdentity(container.Name, utils.GetBizVersionFromContainer(container))
 }
 
-func (mqttTunnel *MqttTunnel) RegisterNode(ctx context.Context, nodeInfo vkModel.NodeInfo) {
+func (mqttTunnel *MqttTunnel) RegisterNode(nodeInfo vkModel.NodeInfo) {
 	nodeID := utils2.ExtractNodeIDFromNodeName(nodeInfo.Metadata.Name)
 	mqttTunnel.mqttClient.Sub(fmt.Sprintf(model.BaseHealthTopic, mqttTunnel.env, nodeID), mqtt.Qos1, mqttTunnel.healthMsgCallback)
```

golangci-lint (1.62.2)

Error return value of `mqttTunnel.mqttClient.Sub` is not checked

(errcheck)